src/oas/__tests__/__snapshots__/operation.test.ts.snap

```diff
@@ -3,6 +3,7 @@
 exports[`oas operation openapi v2 1`] = `
 Array [
   Object {
+    "deprecated": false,
```

Comment:
Kind of feel that these opt-in props, where the default is false (deprecated, internal, required, explode, etc) - perhaps best to just leave them off rather than assign this default value of false? Basically only add them when they're true, to keep the doc size and general "noise" down 🤔. Or maybe it doesn't matter 🤷‍♂️

Reply:
Yeah, I was torn when it comes to truly boolean-ish values like deprecated or internal.
I mostly wanted to avoid inserting invalid values like numbers, strings, etc.
As things stand, we do the following:

```
internal: object['x-internal'] // or deprecated, etc.
```
which is awful since there's no guarantee that that value will be authored as a boolean in the document.
They should be booleans for most of the time, particularly when one uses Studio, but you never know. YAML + code editor... :D



> where the default is false
with `explode` and some others I don't recall now (probably style, so not a boolean) it's actually not that obvious, as the default tends to differ based on some other value.
https://github.com/stoplightio/http-spec/blob/d0be4880b74047e5a3b0ee859a7f10483217433a/src/oas3/transformers/content.ts#L49
<img width="851" alt="image" src="https://user-images.githubusercontent.com/9273484/160466137-9b74a318-09bf-4605-9224-1e6bf9770897.png">

I forgot to include that in the list of changes, but I also considered that lack of normalization as a bug, because I don't think we handled that elsewhere in our tooling.

Reply:
But basically we're storing the "default" value as if it's the user's choice. The "lack of a value" is a specific choice, basically "i want to use the default behavior". By always storing a concrete value, we lose the ability to understand if the user wants/wanted to use the default, or they specifically wanted a particular value. 🤔

Reply:
hmmmm, that is correct, although the reality is that our end tooling doesn't really figure out these defaults. 
It's usually "take what http-spec gives us" kind of a situation, so the more we do it in http-spec, the better the support is.

I'm torn, though. Feels like we could actually leave these values unset if not explicitly authored.
If authored, we should pick only when it's a valid value, i.e. boolean.
What do you think?